src/mscorlib/corefx/SR.cs

```diff
@@ -815,4 +815,19 @@ internal static string Arg_UnauthorizedAccessException
     {
         get { return Environment.GetResourceString("Arg_UnauthorizedAccessException"); }
     }
+
+    internal static string ArgumentOutOfRange_GenericPositive
+    {
+        get { return Environment.GetResourceString("ArgumentOutOfRange_GenericPositive"); }
```

Comment:
The plan is switch this file to be auto-generated like everywhere else(https://github.com/dotnet/coreclr/issues/9474)